packages/change-codemod/transforms/add-require-for-global-identifier.js

```diff
@@ -2,13 +2,23 @@
 // top of the file.  Useful when needing to add requires for global identifiers.
 //
 // The identifier must be specified using the --identifier option, e.g. --identifier=sinon
+// The argument for the new require statement can be specified separately from the targeting identifier, e.g. --identifier=_ --packageName=lodash
```

Comment:
Wanted to call the package param `package`, but `package` is a reserved word in JS. So, I'm using `packageName` instead